src/App.tsx

```diff
@@ -1,20 +1,75 @@
-import { useSelector } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
 
+import { useEffect, useState } from 'react';
 import './App.scss';
-import { Start } from './components/Start';
-
-import { isLoading, getMessage } from './store';
+import './styles/general.scss';
+import { getUsersSelector } from './store/selectors';
+import { getUsers } from './helpers/api';
+import { setUsers } from './store';
+import { PostsList } from './components/PostList/PostList';
+import { PostDetails } from './components/PostDetails/PostDetails';
 
 const App = () => {
-  const loading = useSelector(isLoading);
-  const message = useSelector(getMessage) || 'Ready!';
+  const dispatch = useDispatch();
+
+  const users = useSelector(getUsersSelector);
+
+  const [selectedUser, setSelectedUser] = useState('');
+  const [selectedPostId, setSelectedPostId] = useState(0);
```

Comment:
I believe you can remove these states
save selected post and user in redux state